src/client/app/utils/helperFunctions.js

```diff
@@ -100,6 +100,24 @@ const dateSortDesc = (ary) => {
     return [...ary].sort((a, b) => new Date(b.playedAt) - new Date(a.playedAt));
 };
 
+const playlistUpdateMessage = (messageKey) => {
```

Comment:
Also if PR https://github.com/marciaga/kffp-admin/pull/36 is no longer needed, can you close it?